wordpress-mobile/WordPress-Android
# Add background for screens while paging through notifs in the Notifications tab

Fixes #12040

This PR adds background to the screens shown during paging through notifs in the Notifications tab.

To test:

1. Go to Notifications.
2. Tap any notification in list.
3. Swipe left/right to page through notifications.
4. Notice entering notification has background.

Review Instructions:
Only 1 reviewer is required. Anyone can review.

## Regression Notes
1. Potential unintended areas of impact  🟢


2. What I did to test those areas of impact (or what existing automated tests I relied on)  🟢


3. What automated tests I added (or what prevented me from doing so)  🟢

PR submission checklist:
